src/content/get-started/advanced-commands-and-concepts.mdx

```diff
@@ -0,0 +1,91 @@
+- `okteto destroy --volumes`: Destroy the persistent volumes created by the development environment. Persistent volumes are a storage resource that persists beyond the lifecycle of the Development Environment. This allows data to survive and be accessible by new Development Applications that may be created later. This could be useful for data for databases or application state while developing.
+- `okteto destroy --all`: Destroy all Development Environments, including resources annotated with `dev.okteto.com/policy: keep`
+
+## Important Concepts
+
+### [Namespaces](core/namespaces.mdx)
+
+Namespaces in Okteto are where Development Environments get deployed. Each developer starts with a personal Namespace which has the same name as their account but can create as many different Namespaces as they need.
+
+A Namespace can be used for varying durations depending on your needs—ranging from a few weeks for developing a feature to just a couple of days for fixing a bug. Namespaces provide a dedicated environment for isolated development, and they can also be [shared among team members](core/namespaces.mdx#sharing-and-collaboration-with-namespaces) to collaborate on work before pushing changes to Git.
+
+However, many developers prefer to use a single namespace as a replacement for their local environment. In this workflow, instead of creating a new namespace for each feature or bug fix, they switch branches within the same namespace to work on different tasks. This approach mirrors the traditional local development process and eliminates the need to reinitialize data for each new namespace.
+
+When switching branches, it's important to ensure your environment is up-to-date. If your changes involve only code, running `okteto up` is sufficient to sync your changes. However, if the branches include changes to manifest files, you'll need to run `okteto deploy` to fully apply the updates.
+
+This flexibility allows you to choose the workflow that best fits your development process, whether it's using separate namespaces for isolation or a single namespace for continuity and efficiency.
+
+### [Development Environments](development/index.mdx)
+
+Development Environments enable developers to create applications in the cloud without having to worry about Kubernetes. Developers write code locally on their machine with the tools they love, and Okteto transparently updates their application on a cloud dev environment **in real-time as you code**!
+
+[Add states of dev environments and how to see their logs. The same applies to each service: how to see the logs, the meaning of the states, how to get the endpoints, …]
+
+### [Development Containers](development/containers/index.mdx)
+
+Development containers are essential for creating isolated and reproducible development environments. Definitions for the Containers look like docker-compose, but run in Kubernetes. Okteto supports two modes for development containers:
```

Comment:
This first paragraph doesn't make to much sense. I think it's missing dev containers and dev environments concepts